IdentityCore/src/broker_operation/response/MSIDDeviceInfo.h

```diff
@@ -55,6 +55,12 @@ typedef NS_ENUM(NSInteger, MSIDPreferredAuthMethod)
     MSIDPreferredAuthMethodQRPIN
 };
 
+typedef NS_ENUM(NSInteger, MSIDSsoProviderType)
+{
+    MSIDUnknownSsoProvider = 0,
+    MSIDMacBrokerSsoProvider,
+    MSIDCompanyPortalSsoProvider
+};
```

Comment:
nit: Should we include iOS Authenticator SSO Extension in case it's needed in the future?

Reply:
We can expand the list for iOS if needed, currently the list and the logic behind is only for MacOS